tests/conf/nccmsentinel2.yaml

```diff
@@ -0,0 +1,18 @@
+model:
+  class_path: SemanticSegmentationTask
+  init_args:
+    loss: "ce"
+    model: "deeplabv3+"
+    backbone: "resnet34"
+    in_channels: 4
+    num_classes: 14
+    num_filters: 1
+    ignore_index: 4
+data:
+  class_path: NCCMSentinel2DataModule
+  init_args:
+    batch_size: 2
+    patch_size: 32
+  dict_kwargs:
+    nccm_paths: "tests/data/nccm"
+    sentinel2_paths: "tests/data/sentinel2"
```

Comment:
Can you add an EOL char before the EOF char? It should be possible to set your editor to do this automatically.